MediaGalleryRenditions/etc/module.xml

```diff
@@ -6,5 +6,9 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_MediaGalleryRenditions" />
+    <module name="Magento_MediaGalleryRenditions">
+        <sequence>
+            <module name="Magento_MediaGallerySynchronizationApi" />
```

Comment:
The sequece is not needed, it should not matter which of these modules is loaded first

Reply:
If I'm not adding the sequence then `ImportFilesComposite` is not executing the `GenerateRenditionImages`